book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,300 @@
+
+For other events that require active participation, such as a focus group or discussions, consider including guidance on the format discussions will have, and if there are any specific topics or prior material attendees should be familiar with in order to take part. 
+Try to identify if your event will facilitate discussion between attendees, or if the sessions are to be chaired and led by a facilitator. 
+Communicating to attendees that participation is expected, and what the specific format of participation will be, can help alleviate worry or anxiety that attendees may have, and also set the session up for success. 
+
+(cl-inclusive-events-engagement-pathways)=
+### Establishing Pathways for Engagement
+People with lived experience may have varying comfort levels with public events, in addition to concerns about how any disabilities have been supported (or not) in past events.
+Consider providing explicit guidelines and options for engagement to ensure that everyone is able to contribute at their own pace and in a manner that suits their preferences and needs. 
+
+For example, a discussion or contribution-based event may consider offering both spoken and written contributions. 
+"Speaking" group(s) can be facilitated by a note taker, and the "Writing" group(s) can work together in collaborative documents. 
+You may also consider recording your event's content and making documents available online as part of supporting asynchronous contributions from people in multiple time zones or who may experience difficulties in contributing during "live" events. 
+
+Consider if you are able to design and support sessions that allow for multiple channels of engagement to be as inclusive of as many accessibility requirements and preferences as possible.
+Because all events are limited by the resources available to plan and host the event, it is important to provide clear guidance on what engagement means for your event and to work with a representative with lived experience to identify what methods of engagement your event will need to support.
+
+Ensure that you provide an outline of your guidance in both a written format before the event (preferably in plain text), and also include an explanation in your event introduction of what engagement is expected, as well as what pathways are available for contributions.
+
+(cl-inclusive-events-defining-goals)=
+### Sharing the Goals of your Event
+Consider outlining and sharing a set of goals or planned outputs from your event with your audience when you begin a session. 
+For example, a workshop may present learning objectives or an example of the planned finished product. 
+
+By clearly articulating the event's objectives, you will be able to create a sense of transparency and purpose for the session. 
+This clarity allows all attendees, including people with lived experience, to understand what to expect from the session and reflexively evaluate whether their participation aligns with those goals. 
+
```

Comment:
```suggestion
Additionally, consider if your goals are measurable so that you can more easily conclude after the event whether the goals were reached. 
```
Or something like that - from the OSFNL crew